### PR TITLE
added receipe for basster\legacy-bridge-bundle

### PR DESCRIPTION
**Pull request by Basster**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: basster/legacy-bridge-bundle/1.0/config/packages/legacy_bridge.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+basster_legacy_bridge:
+    # change path to the folder containing your legacy .php scripts.
+    legacy_path:    '%kernel.root_dir%'
+    # optional prepend script (see http://php.net/manual/en/ini.core.php#ini.auto-prepend-file)
+    # prepend_script: '/full/path/to/my/legacy/autoPrependFile.php' # can be ommited
```
Comment (Nyholm) [inline basster/legacy-bridge-bundle/1.0/config/packages/legacy_bridge.yaml]: Im not a big fan of config in comments like this. Could you remove this and add a link to the configuration instead?

File: basster/legacy-bridge-bundle/1.0/config/packages/legacy_bridge.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+basster_legacy_bridge:
+    # change path to the folder containing your legacy .php scripts.
+    legacy_path:    '%kernel.root_dir%'
```
Comment (Nyholm) [inline basster/legacy-bridge-bundle/1.0/config/packages/legacy_bridge.yaml]: Is this a good default?

Comment (OskarStark) [reply basster/legacy-bridge-bundle/1.0/config/packages/legacy_bridge.yaml]: no 👎

File: basster/legacy-bridge-bundle/1.0/config/packages/legacy_bridge.yaml (hunk 1 above)
Comment (OskarStark) [reply basster/legacy-bridge-bundle/1.0/config/packages/legacy_bridge.yaml]: otherwise move this information to your `Configuration` in an `->info()` node

**Comment by Nyholm** [comment]
Closing due inactivity. Feel free to reopen if you want to continue the work. 

#eu-fossa

**Pull request closed without merging**